freqtrade/persistence/migrations.py

```diff
@@ -47,6 +47,13 @@ def migrate_trades_table(decl_base, inspector, engine, table_back_name: str, col
     min_rate = get_column_def(cols, 'min_rate', 'null')
     sell_reason = get_column_def(cols, 'sell_reason', 'null')
     strategy = get_column_def(cols, 'strategy', 'null')
+
+    leverage = get_column_def(cols, 'leverage', '1.0')
+    interest_rate = get_column_def(cols, 'interest_rate', '0.0')
+    isolated_liq = get_column_def(cols, 'isolated_liq', 'null')
+    # sqlite does not support literals for booleans
+    is_short = get_column_def(cols, 'is_short', '0')
+    interest_mode = get_column_def(cols, 'interest_mode', 'null')
     buy_tag = get_column_def(cols, 'buy_tag', 'null')
```

Comment:
I would've left this line "on top" - simply to have them in "order they were added" ... but that's not really relevant.

You will however need to adjust `if not has_column(cols, 'buy_tag'):` in line 174 of this file to one of the new columns to properly trigger migrations - otherwise they'll simply not run if someone has the buy_tag column, but not the "margin" columns - as these columns are the new ones.

Reply:
I don't really understand what you mean by the second part, but I moved buy_tag up